[2025-01-07T00:32:56.987+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-07T00:32:57.003+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ico_scraping_dag_l11.scrape_ico_data manual__2025-01-07T00:20:11.522056+00:00 [queued]>
[2025-01-07T00:32:57.011+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ico_scraping_dag_l11.scrape_ico_data manual__2025-01-07T00:20:11.522056+00:00 [queued]>
[2025-01-07T00:32:57.011+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 3
[2025-01-07T00:32:57.025+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): scrape_ico_data> on 2025-01-07 00:20:11.522056+00:00
[2025-01-07T00:32:57.036+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ico_scraping_dag_l11', 'scrape_ico_data', 'manual__2025-01-07T00:20:11.522056+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/selenium_dag_01.py', '--cfg-path', '/tmp/tmpsklv4ycd']
[2025-01-07T00:32:57.039+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask scrape_ico_data
[2025-01-07T00:32:57.039+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=878) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-07T00:32:57.040+0000] {standard_task_runner.py:72} INFO - Started process 886 to run task
[2025-01-07T00:32:57.084+0000] {task_command.py:467} INFO - Running <TaskInstance: ico_scraping_dag_l11.scrape_ico_data manual__2025-01-07T00:20:11.522056+00:00 [running]> on host e0f05d443e24
[2025-01-07T00:32:57.157+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ico_scraping_dag_l11' AIRFLOW_CTX_TASK_ID='scrape_ico_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T00:20:11.522056+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T00:20:11.522056+00:00'
[2025-01-07T00:32:57.159+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-07T00:32:57.173+0000] {logging_mixin.py:190} INFO - Starting the web scraping process...
[2025-01-07T00:32:57.227+0000] {logging_mixin.py:190} INFO - Chrome version: Chromium 114.0.5735.0
[2025-01-07T00:32:57.228+0000] {logging_mixin.py:190} INFO - ChromeDriver version: ChromeDriver 114.0.5735.90 (386bc09e8f4f2e025eddae123f36f6263096ae49-refs/branch-heads/5735@{#1052})
[2025-01-07T00:32:57.228+0000] {logging_mixin.py:190} INFO - Chrome binary location set to: /opt/chrome/chrome
[2025-01-07T00:32:57.229+0000] {logging_mixin.py:190} INFO - ChromeDriver path set to: /opt/chromedriver/chromedriver
[2025-01-07T00:32:57.361+0000] {logging_mixin.py:190} INFO - WebDriver initialized successfully.
[2025-01-07T00:32:57.361+0000] {logging_mixin.py:190} INFO - Navigating to the ICO Drops page...
[2025-01-07T00:32:58.407+0000] {logging_mixin.py:190} INFO - Attempting to click element with xpath: //button[contains(text(), 'Agree')]
[2025-01-07T00:33:11.408+0000] {logging_mixin.py:190} INFO - Consent button not found or already agreed.
[2025-01-07T00:33:14.428+0000] {logging_mixin.py:190} INFO - Found 100 ICOs on the page.
[2025-01-07T00:33:14.429+0000] {logging_mixin.py:190} INFO - 
Processing ICO 1/100
[2025-01-07T00:33:14.453+0000] {logging_mixin.py:190} INFO - Extracted project_name: Beam
[2025-01-07T00:33:14.480+0000] {logging_mixin.py:190} INFO - Extracted project_ticker: BEAM
[2025-01-07T00:33:14.499+0000] {logging_mixin.py:190} INFO - Extracted project_link: https://icodrops.com/beam/
[2025-01-07T00:33:14.529+0000] {logging_mixin.py:190} INFO - Extracted project_round: Node Sale Round
[2025-01-07T00:33:14.556+0000] {logging_mixin.py:190} INFO - Extracted project_total_raised: $77.75 M
[2025-01-07T00:33:14.586+0000] {logging_mixin.py:190} INFO - Extracted project_pre_valuation: $1.77 B
[2025-01-07T00:33:14.624+0000] {logging_mixin.py:190} INFO - Extracted project_categories: Gaming
[2025-01-07T00:33:14.657+0000] {logging_mixin.py:190} INFO - Extracted project_roi: 0.01x
[2025-01-07T00:33:14.682+0000] {logging_mixin.py:190} INFO - Extracted project_date: Q4, 2024
[2025-01-07T00:33:14.756+0000] {logging_mixin.py:190} INFO - Extracted project_total_investors: 11
[2025-01-07T00:33:14.757+0000] {logging_mixin.py:190} INFO - 
Processing ICO 2/100
[2025-01-07T00:33:14.785+0000] {logging_mixin.py:190} INFO - Extracted project_name: aiPump
[2025-01-07T00:33:14.808+0000] {logging_mixin.py:190} INFO - Extracted project_ticker: AIPUMP
[2025-01-07T00:33:14.834+0000] {logging_mixin.py:190} INFO - Extracted project_link: https://icodrops.com/aipump/
[2025-01-07T00:33:14.856+0000] {logging_mixin.py:190} INFO - Extracted project_round: IDO on Polkastarter
[2025-01-07T00:33:14.885+0000] {logging_mixin.py:190} INFO - Extracted project_total_raised: —
[2025-01-07T00:33:14.911+0000] {logging_mixin.py:190} INFO - Extracted project_pre_valuation: $6 M  
[2025-01-07T00:33:14.937+0000] {logging_mixin.py:190} INFO - Extracted project_categories: AI Agents
[2025-01-07T00:33:14.964+0000] {logging_mixin.py:190} INFO - Extracted project_roi: —
[2025-01-07T00:33:14.990+0000] {logging_mixin.py:190} INFO - Extracted project_date: in 12h
[2025-01-07T00:33:20.174+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-01-07T00:33:20.175+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-07T00:33:20.177+0000] {process_utils.py:132} INFO - Sending 15 to group 886. PIDs of all processes in the group: [892, 898, 905, 906, 907, 886]
[2025-01-07T00:33:20.177+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 886
[2025-01-07T00:33:20.177+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-01-07T00:33:20.179+0000] {connectionpool.py:870} WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f000c2bb320>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/434e60d4aa04c89e054fe016ec370bfc
[2025-01-07T00:33:20.180+0000] {connectionpool.py:870} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f000c0f5a00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/434e60d4aa04c89e054fe016ec370bfc
[2025-01-07T00:33:20.180+0000] {connectionpool.py:870} WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f000c0f5a30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/434e60d4aa04c89e054fe016ec370bfc
[2025-01-07T00:33:20.229+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-07T00:33:20.229+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=892, status='terminated', started='00:32:56') (892) terminated with exit code None
[2025-01-07T00:33:20.230+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=898, status='terminated', started='00:32:56') (898) terminated with exit code None
[2025-01-07T00:33:20.230+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=905, status='terminated', started='00:32:56') (905) terminated with exit code None
[2025-01-07T00:33:20.230+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=906, status='terminated', started='00:32:56') (906) terminated with exit code None
[2025-01-07T00:33:20.231+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=907, status='terminated', started='00:32:56') (907) terminated with exit code None
[2025-01-07T00:33:20.238+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=886, status='terminated', exitcode=0, started='00:32:56') (886) terminated with exit code 0
